src/twisted/web/http.py

```diff
@@ -1236,6 +1239,13 @@ def write(self, data):
 
             for name, values in self.responseHeaders.getAllRawHeaders():
                 for value in values:
+                    if name == b"Content-Length" and isOkay and isCONNECT:
+                        self._log.info(
+                            "Warning: Content-Length header was"
+                            " sent in response to CONNECT request."
+                            " Dropping it."
```

Comment:
Maybe I don't understand what is going in here :)

My understanding is that the header was received by us / the Twisted based app.

It was sent by the remote HTTP proxy server.  

```suggestion
                            "Warning: Content-Length header was"
                            " receive in response to CONNECT request."
                            " Dropping it."
```

Reply:
Just corrected the tense in the suggestion, but I don't think it's necessary inaccurate - somebody's always sending, someone's receiving.